src/content/docs/style-guide/formatting/code-examples.mdx

```diff
@@ -89,22 +89,47 @@ These sections break down when to use specific types of code formatting.
 <CollapserGroup>
   <Collapser
     id="inline-code"
-    title={<>For inline code or data (<InlineCode>&lt;code></InlineCode>)</>}
+    title="Inline code"
   >
-    To surround small blocks of code or data (single words or lines), mark as code:
-
-    * **Source**: Use `<code>` tags.
-    * **Markdown**: Surround the text with backtick `` ` `` characters.
+    To format small code snippets within a sentence, surround the code with single backtick ( `` ` `` ) characters. For example, you might be talking about the `app_name` parameter in a sentence about agent configuration.
   </Collapser>
 
   <Collapser
     id="code-block"
-    title={<>For multi-line code blocks (<InlineCode>&lt;pre></InlineCode>)</>}
+    title="Code blocks"
   >
-    To surround longer blocks of data that run multiple lines, mark as preformatted text:
-
-    * **Source**: Use `<pre>` tags.
-    * **Markdown**: Do not use Markdown-style indented code formatting, as this can cause unexpected formatting problems.
+  To format one or more lines of code, insert three backticks ( `` ``` `` ) above and below your code. This essentially acts like an HTML &lt;pre&gt; tag.
+
+  To add syntax highlighting to a code block, insert the language name following the first three backtacks. Here's an example with handlebars formatting:
+
+  ![Screenshot showing how to format a handlebars code block](./images/code-examples.png "Screenshot showing how to format a handlebars code block")
+
+  This is what the formatting looks like when it's published:
+
+```handlebars
+{{json data.tags}}
+```
+
+  To figure out if the language you want to format is already set up in Gatsby, see if it's listed in `gatsby-config.js` under `prism`. The list of currently supported languages would look like this:
```

Comment:
I'd be more straight forward.

Check in `gatsby-config.js` (under `prism`) if the language you want to format is already set up in Gatsby. The list of currently supported languages is:

Reply:
Thanks, @urbiz-nr !